### PR TITLE
Release 1.0.0

### LINKED ISSUE #35: Get it under the voxpupuli umbrella
We agreed on trying to submit the module under the voxpupuli umbrella

https://voxpupuli.org/docs/migrate_module/

@traylenator seems to have some experience with it.

### PR DESCRIPTION
**Pull request by traylenator**
#### Release 1.0.0
First release to puppetforge 

#### This Pull Request (PR) fixes the following issues
Fixes #35

### CODE REVIEW

**Comment by traylenator** [comment]
Submitting to add tags to old MRs and decide what we want prior to a release.

What happened to the no changelog tag?

**Comment by bastelfreak** [comment]
should we merge https://github.com/voxpupuli/puppet-nftables/pull/42 and the other grene PRs before a release? otherwise we soon need to do a 2.0.0 release.

**Comment by nbarrientos** [comment]
Yup, it'd be nice if https://github.com/voxpupuli/puppet-nftables/pull/47 and https://github.com/voxpupuli/puppet-nftables/pull/48 made it, too.

**Comment by traylenator** [comment]
Just noticed that nftables::set was not documented so added #55  but thing we are there for a release then.

**Review by bastelfreak — APPROVED** [review]

**Comment by traylenator** [comment]
@duritong okay to release?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,76 @@
...
+## [v1.0.0](https://github.com/voxpupuli/puppet-nftables/tree/v1.0.0) (2020-12-14)
 
-**Features**
+[Full Changelog](https://github.com/voxpupuli/puppet-nftables/compare/0ba57c66a35ed4e9b570d8a6315a33a1c4ba3181...v1.0.0)
 
-**Bugfixes**
+**Breaking changes:**
 
-**Known Issues**
+- switch not the server naming [\#42](https://github.com/voxpupuli/puppet-nftables/pull/42) ([duritong](https://github.com/duritong))
```
Comment (duritong) [inline CHANGELOG.md]: oh my typo made it into the release notes, maybe fix that?

Comment (duritong) [reply CHANGELOG.md]: oh and why is this a known issue?

**Review by duritong — APPROVED** [review]

**Comment by duritong** [comment]
minor typo and question about the classification otherwise, let's make 1.0!

File: CHANGELOG.md (hunk 1 above)
Comment (traylenator) [reply CHANGELOG.md]: It's better to fix the resolved MR and regenerate CHANGELOG as it is regenerated from scratch every time.

Looking up the other question.

Comment (traylenator) [reply CHANGELOG.md]: Added the enhancement tag also which I guess will remove it to a normal category.

**Comment by traylenator** [comment]
https://forge.puppet.com/modules/puppet/nftables  and done with "travis_release"

Job well done @nbarrientos @duritong @keachi

**Pull request merged**